waspc/waspc.cabal

```diff
@@ -546,13 +547,17 @@ library cli-lib
     , tar ^>=0.5.1.1
     , zlib ^>=0.6.3.0
     , temporary ^>=1.3
-  other-modules: Paths_waspc
+  other-modules:
+    Paths_waspc
+    Wasp.Cli.Command.BuildStart.Client
+    Wasp.Cli.Command.BuildStart.Server
```

Comment:
I added this to `other-modules` because they are supposed to be internal, right?

Reply:
Yeaaaaah kind of, I think we didn't care much because it is a CLI anyway, not a library we post on Hackage. So we were just putting everything under exposed-modules. While we probably should have put most of it under other-modules.

Maybe just put these also under exposed-modules, for consistency? And if we ever feel we want to fix that, we can fix it at one go, instead of somebody wondering why some stuff is under one place and other stuff under another.